crud.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy.orm import Session
+import models
+import schemas
+
+
+def create_author(db: Session, author: schemas.AuthorCreate):
+    db_author = models.Author(**author.dict())
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+    return db_author
+
+
+def get_author(db: Session, author_id: int):
+    return db.query(models.Author).filter(models.Author.id == author_id).first()
+
+
+def get_authors(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(models.Author).offset(skip).limit(limit).all()
+
+
+def update_author(db: Session, author_id: int, author: schemas.AuthorCreate):
```

Comment:
Consider using `schemas.AuthorUpdate` instead of `schemas.AuthorCreate` for the `update_author` function if you have a separate schema for updates. This would allow partial updates and better reflect the intention of updating an existing record.